hugolib/page.go

```diff
@@ -1456,7 +1456,7 @@ func (p *Page) updateMetaData(frontmatter map[string]interface{}) error {
 
 	if draft != nil && published != nil {
 		p.Draft = *draft
-		p.s.Log.WARN.Printf("page %q has both draft and published settings in its frontmatter. Using draft.", p.File.Path())
+		p.s.Log.INFO.Printf("page %q has both draft and published settings in its frontmatter. Using draft.", p.File.Path())
```

Comment:
Should this still be WARN? I don't see a valid use case to allow this. The point of WARN is that people see this issue, and fix it, while still not stopping their site build.

Reply:
Yes.